### PR TITLE
Various little cleanups; idapython comment API

### PR DESCRIPTION
**Pull request by neuschaefer**
Hi, this patch set is mostly cosmetics, but also one functional change: I implemented `get_cmt`/`set_cmt` so disassembler plugins can manipulate comments in the disassembly.

Please review,
thanks.

### CODE REVIEW

File: idaapi.py [hunk 1]
```diff
@@ -376,3 +376,16 @@ def set_processor(p):
 def set_address_space(aspace):
     global ADDRESS_SPACE
     ADDRESS_SPACE = aspace
+
```
Comment (neuschaefer) [inline idaapi.py]: I don't know if this is the right place in the file to add these functions.

Comment (pfalcon) [reply idaapi.py]: The idea is that "interfacing" code section is at the very bottom, so please move your additions above it.

**Comment by pfalcon** [comment]
Thanks for the pull request. Few quick generic comments: a) in the future, please avoid putting unrelated patches in one PR; b) please follow existing pattern for commit messages, it can be easily discerned from git log: https://github.com/pfalcon/ScratchABit/commits/master

File: idaapi.py (hunk 1 above)
Comment (neuschaefer) [reply idaapi.py]: Ok, I'll update that commit.

**Comment by neuschaefer** [comment]
> Thanks for the pull request. Few quick generic comments:
> a) in the future, please avoid putting unrelated patches in one PR;

Ok

> b) please follow existing pattern for commit messages, it can be easily discerned from git log: https://github.com/pfalcon/ScratchABit/commits/master

Ok, I'll update the commits to follow this pattern.

**Comment by pfalcon** [comment]
Thanks, all is merged. But for reference, "engine: save_addr_props: fix for loop indentation" and "curses: Remove directory" are maintainer-level changes. In this case, these changes were per the plan (somewhat backlogged/forgotten plan), but I may be reluctant to apply similar changes if they "unexpected", with the argument e.g. that they introduce git blame noise ;-). So, it's recommended to discuss similar changes in the future.

**Comment by neuschaefer** [comment]
> [...] So, it's recommended to discuss similar changes in the future.

Ok, fair comments. I'll consider it in the future.

**Pull request merged**